orion/sites/admin/src/programs/program.js

```diff
@@ -22,24 +22,52 @@ export class ProgramPage extends React.Component {
       });
   }
 
-	render() {
+  onClickSelectAll() {
+      var items = document.getElementsByName("unpublished");
+      for (var i=0; i<items.length; i++) {
+          if (items[i].type == 'checkbox') {
+              items[i].checked = true;
+          }
+      }
+  }
+
+  render() {
     const rows = this.state.list.map((row, index) => {
       return <ProgramRow key={index} row={row}/>
     });
     const numRows = rows.length;
-		return (
+    let numUnpublished = 0;
+    let numSelected = 0;
+	return (
       <div id="view-program">
-        <h1>All Programs ({numRows})</h1>
+        <ul>
+            <h1>All Programs ({numRows}) </h1>
+            <div> You have {numUnpublished} unpublished items. </div>
```

Comment:
Maybe you could bundle both of these `<div>s` into a single `<p>`. `<p>` tag is for paragraphs.